### PR TITLE
Updating options on GH-Actions-Importer with new ADO options

### PR DESCRIPTION
**Pull request by begonaguereca**
## What's changing?
Make sure you have the latest version of the Valet container: 
`gh actions-importer update`

Build c# project:
`dotnet build src/ActionsImporter/ActionsImporter.csproj `

Make sure you add a sample ADO pipeline to `output/pipeline.yml`
```
trigger:
- main

pool:
  vmImage: ubuntu-latest

steps:
- script: echo Hello, world!
  displayName: 'Run a one-line script'

- script: |
    echo Add other tasks to build, test, and deploy your project.
    echo See https://aka.ms/yaml
  displayName: 'Run a multi-line script'
```

also `output/config.yaml`:
```
pipelines:
  - project_slug: valet-testing/unit/basic-pipeline-sample
    source_file_path: output/pipeline.yml
  - project_slug: valet-testing/unit/directed-acrylic-graph-pipeline
    source_file_path: output/pipeline.yml

```

Regular commands: 
`dotnet run --project src/ActionsImporter/ActionsImporter.csproj -- dry-run azure-devops pipeline --pipeline-id 427 -o "output"` 

`dotnet run --project src/ActionsImporter/ActionsImporter.csproj -- migrate azure-devops pipeline --pipeline-id 427 -o "output" --target-url "https://github.com/begonaguereca/demo"`

`dotnet run --project src/ActionsImporter/ActionsImporter.csproj -- audit azure-devops -o "output"`
________________________________

Commands with config options:

Run a dry-run with `source-file-path`:
`dotnet run --project src/ActionsImporter/ActionsImporter.csproj -- dry-run azure-devops pipeline -o "output" --source-file-path output/pipeline.yml`

Run for migrate with `source-file-path`: 
`dotnet run --project src/ActionsImporter/ActionsImporter.csproj -- migrate azure-devops pipeline -o "output" --source-file-path output/pipeline.yml --target-url "https://github.com/begonaguereca/demo"`

Run for audit with a `config-file-path`: 
`dotnet run --project src/ActionsImporter/ActionsImporter.csproj-- audit azure-devops -o "output" --config-file-path "output/config.yml" `

## How's this tested?

Closes [related issues]

### CODE REVIEW

File: src/ActionsImporter/Commands/AzureDevOps/Audit.cs [hunk 1]
```diff
@@ -13,6 +13,12 @@ public Audit(string[] args)
     protected override string Name => "azure-devops";
     protected override string Description => "An audit will output a list of data used in an Azure DevOps instance.";
 
+    private static readonly Option<FileInfo> ConfigFilePath = new("--config-file-path")
```
Comment (j-dunham) [inline src/ActionsImporter/Commands/AzureDevOps/Audit.cs:16]: Should this be added to the `Options`?

Comment (begonaguereca) [reply src/ActionsImporter/Commands/AzureDevOps/Audit.cs:16]: woff had it in Valet but not here! good catch

File: src/ActionsImporter/Commands/AzureDevOps/BuildPipeline/DryRun.cs [hunk 2]
```diff
@@ -11,5 +11,15 @@ public DryRun(string[] args) : base(args)
 
     protected override string Name => "pipeline";
     protected override string Description => "Target a designer or YAML pipeline";
-    protected override ImmutableArray<Option> Options => ImmutableArray<Option>.Empty;
+
+    public static readonly Option<int> PipelineId = new(new[] { "--pipeline-id", "-i" })
```
Comment (j-dunham) [inline src/ActionsImporter/Commands/AzureDevOps/BuildPipeline/DryRun.cs:15]: should the changes in this file be applied to `src/ActionsImporter/Commands/AzureDevOps/DryRun.cs` instead and be reverted here?

Comment (begonaguereca) [reply src/ActionsImporter/Commands/AzureDevOps/BuildPipeline/DryRun.cs:15]: no because that is for both build _and_ release no?

Comment (j-dunham) [reply src/ActionsImporter/Commands/AzureDevOps/BuildPipeline/DryRun.cs:15]: oh yeah I thought that was a valid option for both.

File: src/ActionsImporter/Commands/AzureDevOps/BuildPipeline/DryRun.cs [hunk 3]
```diff
@@ -11,5 +11,15 @@ public DryRun(string[] args) : base(args)
 
     protected override string Name => "pipeline";
     protected override string Description => "Target a designer or YAML pipeline";
-    protected override ImmutableArray<Option> Options => ImmutableArray<Option>.Empty;
+
+    public static readonly Option<int> PipelineId = new(new[] { "-i", "--pipeline-id" })
+    {
+        Description = "The Azure DevOps pipeline id.",
```
[the remark this answers is not in the document]
Comment (ethanis) [reply src/ActionsImporter/Commands/AzureDevOps/BuildPipeline/DryRun.cs:17]: We use lowercase `id` pretty consistently throughout the rest of our docs (e.g. in our [labs](https://github.com/actions/importer-labs/blob/main/azure_devops/4-dry-run.md)).

@JenniferKerns can we add this to the Valet/:chicken: style guide?

File: src/ActionsImporter/Commands/AzureDevOps/Common.cs [hunk 4]
```diff
@@ -28,9 +28,15 @@ public static class Common
         IsRequired = false,
     };
 
-    public static readonly Option<int> PipelineId = new(new[] { "--pipeline-id", "-i" })
+    public static readonly Option<int> PipelineId = new(new[] { "-i", "--pipeline-id" })
```
Comment (ethanis) [inline src/ActionsImporter/Commands/AzureDevOps/Common.cs:31]: Does the ordering of these options address #22 ?

Comment (begonaguereca) [reply src/ActionsImporter/Commands/AzureDevOps/Common.cs:31]: Sorry pulling this into draft, going to get all the changes in Valet first get the approvals and then mirror them over here. I was swaping them for an unrelated issue I was having with pipeline id but let me snag that bug!

**Review by j-dunham — APPROVED** [review]

**Pull request merged**